src/components/FileInput/FileInput.tsx

```diff
@@ -0,0 +1,90 @@
+import { PlusLine } from '@mochi-ui/icons'
+import { useRef, useState } from 'react'
+
+type FileInputProps = {
+  format?: string[]
+  maxSize?: number
+  isMultiple?: boolean
+  onFilesSelect: (files: FileList) => void
+}
+
+const FileInput: React.FC<FileInputProps> = ({
+  format = ['.csv'],
+  maxSize = 100000000,
+  isMultiple,
+  onFilesSelect,
+}: FileInputProps) => {
+  const [files, setFiles] = useState<FileList | null>(null)
+  const fileInputRef = useRef<HTMLInputElement>(null)
+
+  // Hàm xử lý khi người dùng chọn file
+  const handleFileSelect = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const selectedFiles = event.target.files
+    if (selectedFiles) {
+      for (const file of selectedFiles) {
+        if (file.size > maxSize) {
+          alert(
```

Comment:
Should use [Toast](https://ds.mochiui.com/?path=/docs/feedback-toast--docs) to show our theming alert